packages/core/demo/demo-data/bar.ts

```diff
@@ -72,7 +72,8 @@ export const groupedBarOptions = {
 		size: "compact"
 	},
 	legendClickable: true,
-	containerResizable: true
+	containerResizable: true,
+	chartTitle: "Bar Chart"
```

Comment:
we can use a simpler key here like `title`, the root level of the options JSON corresponds to the chart anyways